lib/tizen_sdk.dart

```diff
@@ -260,6 +260,11 @@ class TizenSdk {
     }
     apiVersion ??= '5.5';
 
+    if (apiVersion == '8.0') {
+      // Note: From tizen-8.0, profile has been integrated into the `tizen`.
+      profile = 'tizen';
```

Comment:
Did you check if the instruction at lines 311-312 is still valid?

Reply:
You mean this command?
`/home/junsu/tizen-studio/package-manager/package-manager-cli.bin install TIZEN-8.0-NativeAppDevelopment-CLI`
it works well in my environment. is there any thing that i have to check more?

Reply:
No problem if it works. Thanks.